packages/components/__tests__/__snapshots__/switch.tests.tsx.snap

```diff
@@ -0,0 +1,19 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Switch render default component if no match was found 1`] = `
+<h1>
+  Default Component
+</h1>
+`;
+
+exports[`Switch should render the first component that match 1`] = `
+<h1>
+  Test Archive Component
+</h1>
+`;
+
+exports[`Switch should return null if no component match 1`] = `
+<h1>
+  Test Blog Component
+</h1>
+`;
```

Comment:
This is returning the `TestBlog` component instead of `null`.

Reply:
Fixed